### PR TITLE
glances, ifconfig, kill, killall, nano, neofetch, pm2, and screen: add Chinese translation

### PR DESCRIPTION
**Pull request by BananaaaKING**
This PR adds **Chinese versions** for the following commands:

- glances
- ifconfig
- kill
- killall
- nano
- neofetch
- pm2
- screen

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** N/A

### CODE REVIEW

File: pages.zh/common/screen.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# screen
+
+> 在远程服务器上保持会话打开。通过单个 SSH 连接管理多个窗口。
+> 类似工具请参阅 `tmux` 和 `zellij`。
+> 更多信息见：https://manned.org/screen.
```
[the remark this answers is not in the document]
Comment (BananaaaKING) [reply pages.zh/common/screen.md]: Sorry, I clicked the wrong button. Could you submit it again

File: pages.zh/common/glances.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# glances
+
+> 一个跨平台的系统监控工具。
+> 更多信息：<https://nicolargo.github.io/glances/>.
+
+- 启动 glances，实时显示系统状态：
+
+`glances`
+
+- 以 Web 服务器模式启动，在浏览器中查看监控数据：
```
Comment (blueskyson) [inline pages.zh/common/glances.md:10]:
```suggestion
- 以网页服务器模式启动，在浏览器中查看监控数据：
```

File: pages.zh/common/glances.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- 以服务器模式启动，允许其他 glances 客户端连接以查看数据：
+
+`glances -s`
+
+- 作为客户端连接到 glances 服务器：
+
+`glances -c {{主机名或IP地址}}`
+
+- 在（Web）服务器模式下启用密码保护：
```
Comment (blueskyson) [inline pages.zh/common/glances.md:22]:
```suggestion
- 在（网页）服务器模式下启用密码保护：
```

File: pages.zh/common/ifconfig.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# ifconfig
+
+> 网络接口配置工具
```
Comment (blueskyson) [inline pages.zh/common/ifconfig.md:3]:
```suggestion
> 网络接口配置工具。
```

File: pages.zh/common/neofetch.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# neofetch
+
+> 命令行系统信息工具，以美观的方式显示系统信息。
+> 更多信息：<https://github.com/dylanaraps/neofetch>.
+
+- 显示系统信息：
+
+`neofetch`
+
+- 在输出中显示或隐藏指定的信息行（`infoname` 是配置文件中的函数名称，例如 memory）：
```
Comment (blueskyson) [inline pages.zh/common/neofetch.md:10]:
```suggestion
- 在输出中显示或隐藏指定的信息行，「信息名称」是配置文件中的函数名称，例如 memory：
```

**Review by blueskyson — APPROVED** [review]

**Pull request closed without merging**